app2/src/lib/queries/channels.svelte.ts

```diff
@@ -0,0 +1,33 @@
+import { createQueryGraphql } from "$lib/utils/queries.ts"
+import { Option, Schema } from "effect"
+import { graphql } from "gql.tada"
+import { chains } from "$lib/stores/chains.svelte.ts"
+import { Channels } from "$lib/schema/channel.ts"
+import { channels } from "$lib/stores/channels.svelte.ts"
+
+export const channelsQuery = () =>
+  createQueryGraphql({
+    schema: Schema.Struct({ v1_ibc_union_channel_recommendations: Channels }),
+    document: graphql(`
+      query Ucs03Channels @cached(ttl: 60) {
+          v1_ibc_union_channel_recommendations(where: {_and: [{version: {_eq: "ucs03-zkgm-0"}}, {destination_chain_id: {_neq: "11155111"}}, {destination_chain_id: {_neq: "17000"}}]}) {
```

Comment:
you can remove the destination_chain filter, it was temporarily there for app v1

Reply:
also asking jurriaan if there is a `v2` api for this instead of `v1_ibc_union`